features/visual_studio/chat/build/ChatLib/ReadMe.txt

```diff
@@ -0,0 +1,29 @@
+========================================================================
```

Comment:
It won't harm if we remove this readme file

Reply:
Same for other README files

Reply:
Done